packages/api/src/@core/@core-services/unification/ingest-data.service.ts

```
@@ -85,10 +85,16 @@
         .filter((p) => p.shouldPassToService)
         .map((p) => p.param);
 
+      const ingestParams = params
+        .filter((p) => p.shouldPassToIngest)
+        .reduce((acc, p) => ({ ...acc, [p.paramName]: p.param }), {});
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)